docs/migration_guide.md

```diff
@@ -74,7 +74,7 @@ For any reason, if you want to roll back to the previous version of Harbor, foll
     cd harbor
     docker-compose down
     ```
-2. Restore database from backup file in `/path/to/backup` .
+2. Restore database from backup file in `/path/to/backup` . **Notice that DB engine is diffrent between 1.2 and 1.3, so rollback from harbor 1.3 to harbor 1.2 must use `vmware/harbor-db-migrator:1.2`.**
```

Comment:
Suggest move to a **NOTE:** section, and reference the issue.  
And fix the typo :)